.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+### Completed each of the following items before creating a PR
+
```

Comment:
Well that's just incorrect. I guess I need to update that documentation. We
don't care what's in the PR description from the pre-check point of view.

GitHub normally populates the PR title and description from the commit
message. I don't know what control we'll have over how that's done.
On Mon, Feb 22, 2016 at 5:09 PM Derek Gaston <notifications@github.com>
wrote:

> In .github/PULL_REQUEST_TEMPLATE.md
> <https://github.com/idaholab/moose/pull/6364#discussion_r53716447>:
>
> > @@ -0,0 +1,6 @@
> > +### Completed each of the following items before creating a PR
> > +
>
> You've said this twice... so I don't understand.... our guidelines say
> that a ticket number must be put in the PR description in addition to the
> commit messages:
>
> (from: http://mooseframework.org/wiki/Contributing/ )
>
> Further, every Pull Request also needs to reference an issue number in both its description.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/6364/files#r53716447>.
>